WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+                                       lineWidth: Layout.borderLineWidth)
+                        Text(Localization.valueRequiredWarningText)
+                            .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                            .footnoteStyle()
+                            .renderedIf(viewModel.weightPerUnit.isEmpty)
+                    }
+                }
+
+                HStack {
+                    Text(Localization.originCountryTitle)
+                        .foregroundColor(.primary)
+                    Spacer()
+                    Button {
+                        // TODO: Add information
+                    } label: {
+                        Image(systemName: "info.circle")
+                            .foregroundColor(Color(.wooCommercePurple(.shade60)))
+
+                    }
+                }
+                    .subheadlineStyle()
+
+                Button {
+                    isShowingCountries = true
+                } label: {
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                            .bodyStyle()
+                        Spacer()
+                        Image(systemName: "chevron.up.chevron.down")
+                    }
+                    .padding()
+                }
+                .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+            }
+            .padding(.leading, Layout.extraPadding)
+            .padding(.trailing, Layout.extraPadding)
+            .padding(.bottom, Layout.extraPadding)
+        })
+        .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+        .safariSheet(isPresented: $isShowingHSTarrifInfoWebView, url: viewModel.hsTariffURL)
+        .sheet(isPresented: $isShowingCountries, content: {
+            NavigationStack {
+                SingleSelectionList(title: Localization.originCountryTitle,
+                                    items: viewModel.allCountries,
+                                    contentKeyPath: \.name,
+                                    selected: $viewModel.originCountry)
+            }
+            .wooNavigationBarStyle()
+        })
+    }
+}
+
+extension WooShippingCustomsItem {
+    enum Localization {
+        static let descriptionTitle = NSLocalizedString("wooShipping.customsItems.description",
+                                              value: "Description",
+                                              comment: "Title for the customs items description text field for customs items")
+        static let HSTarriffNumber = NSLocalizedString("wooShipping.customsItems.hsTarriffNumber",
+                                                       value: "HSTarriffNumber",
```

Comment:
This is missing its spaces, and I think there's a typo in the design. It should be:

```suggestion
                                                       value: "HS tariff number",
```

Reply:
Done in 6915c9c51787e25995858265ec5d5bd06c105400